src/shared/UserStatus.ts

```diff
@@ -0,0 +1,12 @@
+export enum UserStatus {
+  Rejected = 'Rejected',
+  Unconfirmed = 'Unconfirmed',
+  Confirmed = 'Confirmed',
+  Declined = 'Declined',
+  Late = 'Late',
+  Waitlisted = 'Waitlisted',
+}
+
+export function isAcceptableStatus(status: UserStatus): boolean {
+  return new Set([UserStatus.Unconfirmed, UserStatus.Confirmed]).has(status);
```

Comment:
Waitlisted as well?

Reply:
done

Reply:
I meant shouldn't waitlist be on this list

Reply:
Oh yeah that sounds like a good idea. The UI would show 2 buttons - send acceptance and send waitlist then.

Reply:
That's still fine, but you can accept someone without putting them on the waitlist.